src/modules/Dashboard/features/Applet/AddUser/AddUserForm/AddUserForm.tsx

```diff
@@ -86,7 +102,7 @@ export const AddUserForm = ({ getInvitationsHandler }: AddUserFormProps) => {
       unregister(nickname);
       unregister(secretUserId);
       unregister(respondents);
-      workspaceNameShowed && register(workspacePrefix, { value: '' });
+      workspaceNameVisible && register(workspacePrefix, { value: workspaceInfo?.name });
```

Comment:
what if workspaceInfo?.name would return false for register?

Reply:
fixed